env0/data_template.go

```diff
@@ -140,6 +146,10 @@ func dataTemplateRead(ctx context.Context, d *schema.ResourceData, meta interfac
 		d.Set("retry_on_destroy_only_when_matches_regex", "")
 	}
 
+	if template.GithubInstallationId != 0 {
```

Comment:
Shouldn't this be:
```suggestion
	if template.GithubInstallationId != nil {
```
?

Reply:
can't use `nil` here since this is number

Reply:
@eranelbaz so that means default value in case template doesn't have GH installation id would be `0`? 🤔

Reply:
The default value of `int` in go is `0`,
so if we don't have `GithubInstallationId` on the tempalte its value should be `0` IIRC

Reply:
no that won't work

Reply:
At least we need to check it because if we create a template with `githubInstallation: 0` it passes validation but later on makes a lot of trouble for the org